Latest develop does not build with openimageio 1.8.17 metadata.remove

```
/home/skinkie/Sources/AliceVision/src/software/pipeline/main_panoramaWarping.cpp:1210:18: error: 'class OpenImageIO_v1_8::ParamValueList' has no member named 'remove'
 1210 |         metadata.remove("Orientation");
      |                  ^~~~~~
/home/skinkie/Sources/AliceVision/src/software/pipeline/main_panoramaWarping.cpp:1211:18: error: 'class OpenImageIO_v1_8::ParamValueList' has no member named 'remove'
 1211 |         metadata.remove("orientation");
      |  
```

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -27,7 +27,7 @@ Dependencies
 
 AliceVision depends on:
 
-* Boost >= 1.60.0
+* Boost >= 1.70.0
 * Eigen >= 3.3.4
 * Ceres >= 1.10.0
 * Flann >= 1.8.4 (internal)
@@ -35,10 +35,9 @@ AliceVision depends on:
 * Coin-or linear programming (Clp) (internal)
 * Open Solver Interface (Osi) >= 0.106.10 (internal)
 * Lemon >= 1.3 (internal)
-* OpenEXR >= 2.2.0
-* OpenImageIO >= 1.8.7
+* OpenEXR >= 2.4.0
+* OpenImageIO >= 2.1.0
 * Geogram >= 1.5.4 (https://gforge.inria.fr/frs/?group_id=5833)
-* OpenEXR >= 2.2
 * MeshSDFilter (internal)
 * OpenMesh (internal)
 * zlib

